ignite/handlers/state_param_scheduler.py

```diff
@@ -329,21 +329,42 @@ class StepStateScheduler(StateParamScheduler):
 
     Examples:
 
-        .. code-block:: python
+        .. testsetup::
+
+            engine = get_default_trainer()
```

Comment:
```suggestion
            default_trainer = get_default_trainer()
```

Sorry, I realized that `default_trainer` would be the engine name. Could you rename in the other examples in `ExpStateScheduler` too ?

Reply:
ok. will do